lib/Common/Core/CommonMinMax.h

```diff
@@ -6,8 +6,8 @@
 
 template<class T> inline
 _Post_equal_to_(a < b ? a : b) _Post_satisfies_(return <= a && return <= b)
-    const T& min(const T& a, const T& b) { return a < b ? a : b; }
+    const T& GET_MIN(const T& a, const T& b) { return a < b ? a : b; }
 
 template<class T> inline
 _Post_equal_to_(a > b ? a : b) _Post_satisfies_(return >= a && return >= b)
-    const T& max(const T& a, const T& b) { return a > b ? a : b; }
+    const T& GET_MAX(const T& a, const T& b) { return a > b ? a : b; }
```

Comment:
nit: the casing kind of implies that this is a macro. I feel like a `GetMax` or just 'Max' would be better

Reply:
@MikeHolman agree. I would gladly consider `GetMax` or `Max` at the first place. Well, such a large project + dependencies. `GET_MAX` is kinda different enough to define at global scale?

Updating the original definitions to `GetM**Global` and re-defining back as `GET_M**`
```
#define GET_MAX GetMaximumGlobal
#define GET_MIN GetMinimumGlobal
```

So, this should work for anyone is searching for `define GET_MAX` project wide